src/Microsoft.Fast.Components.FluentUI/ColumnDefinition.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace Microsoft.Fast.Components.FluentUI
+{
+	public class ColumnDefinition<TItem>
+	{
+		public string ColumnDataKey { get; set; }
+		public string Title { get; set; }
+
+		public string HeaderCellTemplate { get; set; }
+		public bool HeaderCellInternalFocusQueue { get; set; }
+		public int HeaderCellFocusTargetCallback { get; set; }
+
+
+		public string CellTemplate { get; set; }
+		public bool CellInternalFocusQueue { get; set; }
+		public int CellInternalFocusTargetCallback { get; }
```

Comment:
Do these do anything?  The underlying type on the JS side is a function.

Reply:
No, bot doing anything atm. I'll remove them.

Reply:
No, they don't do anything. I'll remove them for now. We can always add them if needed.